opba-facade-protocol-api-shared/src/main/java/de/adorsys/opba/protocol/api/services/scoped/validation/Rules.java

```diff
@@ -0,0 +1,5 @@
+package de.adorsys.opba.protocol.api.services.scoped.validation;
+
+public interface Rules {
```

Comment:
IgnoreValidationRule or SkipValidationRule seem to be more precise name based on its usage in Xs2aValidator